src/option.c

```diff
@@ -61,6 +61,7 @@
 #define PV_BT		OPT_BUF(BV_BT)
 #ifdef FEAT_QUICKFIX
 # define PV_EFM		OPT_BOTH(OPT_BUF(BV_EFM))
+# define PV_RNFM		OPT_BOTH(OPT_BUF(BV_RNFM))
```

Comment:
You put this line inside `#ifdef FEAT_QUICKFIX`, however this is not a part of the quickfix feature.
It should be moved to outside of the `#ifdef`.

Additionally, you implemented this as "global or local to buffer" (`:help global-local`), but I think it should be "local to window" (as I wrote above).
Please check the implementation of 'rnu'.